### DISCUSSION TITLE
TypeAlias error

### DISCUSSION THREAD

**Discussion by zb-saml** [jupyterlite/jupyterlite#1381] [General]
Dear community members, 

When I started developing in Jupyterlite, it worked until around Jan 2024, but from that point on forwards I was not able to run Jupyter Notebooks on Jupyterlite anymore. The reason for this was that it was throwing a TypeAlias error, meaning it could not find the package of the method TypeAlias. One week ago, I have tried to run JupyterNotebooks on Jupyterlite again and it worked fine without me trying to debug the error. For the sake of reliability and stability, I would like to ask why this error occurred in the first place and how it was fixed ever since? 

Thank you in advance for sharing this piece of information. 

Kind regards, 
zb-saml

**Comment by jtpio** [comment]
Thanks @zb-saml for the report.

Would you be able to share a screenshot of the error to help identify what this `TypeAlias` error could be?
